amarachukwu-agbo/random-number-generator
# #165145298 Sort numbers

#### What does this PR do?
Enables sorting of numbers

#### Description of Task to be completed?
- Add component for sorting numbers
- Write tests
- Add styles

#### How should this be manually tested?
- Clone the repo and install application dependencies. 
- Run command  `yarn run client:start-dev`. 
- On another terminal, run command  `yarn run server:start-dev`. 
- Click on any of the sort icons. The numbers should be sorted based on the selected icon

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#165145298](https://www.pivotaltracker.com/story/show/165145298)

#### Screenshots (if appropriate)
N/A

#### Questions: 
N/A
